src/mixins/date-mixin.ts

```diff
@@ -28,9 +28,12 @@ export default class DateMixin extends Vue {
    * @returns the formatted date string
    */
   toReadableDate (date: string): string {
-    // cast to a workable dateString
+    // cast to a workable dateString 'America/Vancouver'
     // split into an array
-    let formatDate = (new Date(date).toDateString()).split(' ')
+    let newDate = new Date(date)
+    // Factor in the UTC offset so date displayed is local
+    newDate.setTime(newDate.getTime() + newDate.getTimezoneOffset() * 60 * 1000)
```

Comment:
This change needs visibility in this commit/PR/ticket (ie, for traceability and external testing).

Also this method is now different from the same method in our other projects. What's the plan to update those?

---

Would it make more sense to have a **single** UTC-to-local method in this mixin (ie, convertUtcTimeToLocalTime () below) and use **this** method to simply do the formatting?

Reply:
I'm not sure if one method will be applicable to all? The dates seem to come in many different formats.

We can/will update this in the other repos as well :) It was an error that would be present in any of its use cases.

Reply:
I'm concerned about 2 methods in this mixin that apparently do something similar (and do it in different ways!).

Reply:
I'm also concerned that the description for this method says the date comes in as YYYY-MM-DD (ie, no time and no timezone)... is it for sure a UTC date? If not then this is dangerous as it could be used on a local date and give an incorrect result.

Reply:
Hmm yea, not sure what to do here. My goal was to correct a bug found during unit testing, but if you think we need to look at this mixin and refactor as a whole we can certainly do that.

Reply:
What if you don't fix it here and instead create a ticket to review/fix this for all 3 projects?

Reply:
That sounds great, we should further discuss the solution!

Reply:
For posterity: We decided to NOT format the current date in the Certify component at this time (since it raised the issues above). This is now consistent with the other 2 UIs. A new ticket will be created to format the current date consistently (and correctly) in all 3 UIs at the same time.